syntax/std_seq_split_test.go

```diff
@@ -0,0 +1,89 @@
+package syntax
+
+import "testing"
+
+func TestStrSplit(t *testing.T) {
+	t.Parallel()
+
+	AssertCodesEvalToSameValue(t, `["this", "is", "a", "test"]`, `//seq.split(" ","this is a test") `)
+	AssertCodesEvalToSameValue(t, `["this is a test"]         `, `//seq.split(",","this is a test") `)
+	AssertCodesEvalToSameValue(t, `["th", " ", " a test"]     `, `//seq.split("is","this is a test")`)
+	assertExprPanics(t, `//seq.split(1, "this is a test")`)
+
+	AssertCodesEvalToSameValue(t,
+		`["t", "h", "i", "s", " ", "i", "s", " ", "a", " ", "t", "e", "s", "t"]`,
+		`//seq.split("","this is a test")`)
+
+	// As https://github.com/arr-ai/arrai/issues/268, `{}`, `[]` and `""` means empty set in arr.ai
+	// And the intent for //seq.split is to return an array, so it should be expressed as such.
+	// `""` -> empty string, `[]` -> empty array and `{}` -> empty set
+	AssertCodesEvalToSameValue(t, `[]`, `//seq.split("","") `)
+
+	AssertCodesEvalToSameValue(t, `[""]`, `//seq.split(",","") `)
+
+	assertExprPanics(t, `//seq.split(1,"ABC")`)
+}
+
+func TestArraySplit(t *testing.T) {
+	t.Parallel()
+	AssertCodesEvalToSameValue(t,
+		`[['B'],['C', 'D', 'E']]`,
```

Comment:
```suggestion
		`[[], ['B'],['C', 'D', 'E']]`,
```
Same at the end. https://play.golang.org/p/dNDCa5M9Krw

The only exception is when `sep` is empty: `//seq.split([],['A', 'B']) = [['A'], ['B']]`.

Reply:
Fixed.